src/app/examples/examples.module.tns.ts

```diff
@@ -2,14 +2,15 @@
 
 import { NgModule, NO_ERRORS_SCHEMA } from '@angular/core';
 import { NativeScriptCommonModule } from 'nativescript-angular/common';
+import { NativeScriptFormsModule } from 'nativescript-angular';
 
 import { KirbyModule } from '../../kirby/kirby.module';
 import { COMPONENT_DECLARATIONS } from './examples.common';
 import { NativeScriptDoughnutChartExampleComponent } from './nativescript-only/doughnut-chart-example/doughnut-chart-example.component.tns-only';
 import { NativeScriptLineChartExampleComponent } from './nativescript-only/line-chart-example/line-chart-example.component.tns-only';
-import { ListSectionExampleComponent } from './list/tns-list-examples/list-section-example/list-section-example.component.tns';
-import { ListCustomCellLinesExampleComponent } from './list/tns-list-examples/list-custom-cell-lines-example/list-custom-cell-lines-example.component.tns';
-import { ListItemExampleComponent } from './list/tns-list-examples/list-item-example/list-item-example.component.tns';
+import { ListItemExampleComponent } from '~/app/examples/list/list-item-example/list-item-example.component.tns';
```

Comment:
Don't use absolute imports - it's easer to grasp that this lives inside the examples folder as does the module

Reply:
Removed, and removed `paths`-element from `tsconfig.json`